zerver/views/webhooks/trello/board_actions.py

```diff
@@ -0,0 +1,107 @@
+from typing import Dict, Any, Tuple
+from .exceptions import UnknownUpdateBoardAction
+from .templates import TRELLO_SUBJECT_TEMPLATE, TRELLO_MESSAGE_TEMPLATE
+
+SUPPORTED_BOARD_ACTIONS = [
+    'removeMemberFromBoard',
+    'addMemberToBoard',
+    'createList',
+    'updateBoard',
+]
+
+REMOVE_MEMBER = 'removeMemberFromBoard'
+ADD_MEMBER = 'addMemberToBoard'
+CREATE_LIST = 'createList'
+CHANGE_NAME = 'changeName'
+
+TRELLO_BOARD_URL_TEMPLATE = '[{board_name}]({board_url})'
+
+ACTIONS_TO_MESSAGE_MAPPER = {
+    REMOVE_MEMBER: 'removed {member_name} from {board_url_template}',
+    ADD_MEMBER: 'added {member_name} to {board_url_template}',
+    CREATE_LIST: 'added {list_name} list to {board_url_template}',
+    CHANGE_NAME: 'renamed the board from {old_name} to {board_url_template}'
+}
+
+def process_board_action(payload, action_type):
+    # type: (Dict[str, Any], str) -> Tuple[str, str]
```

Comment:
This should return a `Tuple[text_type, text_type]`, since subject and body can have non-ASCII characters. Also `action_type` should be `text_type`.